Maker-Agents-Suit-Module/ESMAgent/src/ESM_Join_Event_Agent.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  FindingGenerator,
+  provideEventCheckerHandler,
+} from '@nethermindeth/general-agents-module';
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  Log,
+} from 'forta-agent';
+
+export const MAKER_ESM_JOIN_EVENT_SIGNATURE = 'Join(address,uint256)';
+export const MKR_DECIMALS = 18;
+
+const filterLog = (log: Log): boolean => {
+  const value = Number(BigInt(log.data)) / 10 ** MKR_DECIMALS;
```

Comment:
Why this double cast?
If you cast to `BigInt` in order to work with a big number I think that when you cast to `Number` you lose the ability of handling the big number. 

Suggestions:
- If working with `Number` is enough (I think it is not) then remove the `BignInt` cast:
`Number(log.data) / 10 ** MKR_DECIMALS`
- Other option is using `BigInt` in both operands:
`BigInt(log.data) / BigInt(10 ** MKR_DECIMALS)`

Maybe is better to change the comparison and work with `MKR * DECIMALS` to avoid the division